### PR TITLE
Suggested edits to README.md update

### PR DESCRIPTION
**Pull request by drewnielson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
 
 ### What will the working group do with nonmember content?
-The working group will do our best to review any updated test processes from submitters, will attempt to make use of any test content, and share results as progress is made.  
+The working group will do our best to review any suggestions from submitters and share results as progress is made.  
 
-### How will the final working group updated evaluation processes be used?
-The harmonized baseline revised section 508 conformance evaluation process for web and non-web interfaces will be a foundation for creating organization-specific, aligned test procedures. The DHS Trusted Tester process update will fully align with the harmonized baseline update.
+### How will the final evaluation processes be used?
+The harmonized Baseline for web and non-web interfaces will be a foundation for creating organization-specific, Baseline-aligned test procedures. The DHS Trusted Tester process update will fully align with the harmonized Baseline update.
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply README.md:32]: Good question; I think "final evaluation processes" just refers to the final version of the Baseline tests. And yes, we should probably just remove reference to Trusted Tester throughout the site/docs.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,40 @@
...
-The working group will post proposed test processes to this [repository](https://github.com/Section508Coordinators/ICTTestingBaseline/) in GitHub as work progresses. The working group will post test cases to GitHub as well. The working group will update items as refinements are made as we move to final of test processes.
+### How you can contribute:
+The working group will post proposed test processes to this [repository](https://github.com/Section508Coordinators/ICTTestingBaseline/) in GitHub as work progresses. The working group will post test cases to GitHub as well. The working group will update items as refinements are made and we move to final of test processes.
 
-You are welcome to monitor group progress, create your own forks of the proposed test processes and propose changes (via pull requests) to improve them, and submit test content to help validate test processes and tools.
+You are welcome to monitor group progress, create your own forks of the proposed test processes, propose changes (via pull requests) to improve them, and submit test content to help validate test processes and tools.
 
 ### What will the working group do with nonmember content?
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply README.md:28]: Yes, agreed.

File: README.md [hunk 3]
```diff
@@ -1,43 +1,40 @@
...
+### How will the final evaluation processes be used?
+The harmonized Baseline for web and non-web interfaces will be a foundation for creating organization-specific, Baseline-aligned test procedures. The DHS Trusted Tester process update will fully align with the harmonized Baseline update.
 
 ### How to contact the working group:
-If you wish to contact the working group with suggestions you can email to accessibility@dhs.gov and include in the subject "Federal Accessibility Community of Practice: Trusted Tester Update."  
+If you wish to contact the working group with suggestions, you can email ictbaseline@gsa.gov and include in the subject "ICT Baseline Update."  
 
 **Note:** the working group makes no commitment to accept or use recommendations, but will do our best to review them and consider them as the update process is completed.
 
 ### What the working group is NOT doing at this time:
```
[the remark this answers is not in the document]
Comment (drewnielson) [reply README.md:39]: Agreed. We probably don't need to make any reference to automation vs. manual here.

**Comment by drewnielson** [comment]
@kengdoj, this pull request includes suggested edits on top of the edits you were already recommending in pull request #212, so feel free to incorporate however is easiest into the other proposed change. Then we can review the overall proposed change to the Readme file.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kengdoj) [reply README.md:32]: Should the question change to "How will the Baseline be used?"

[the remark this answers is not in the document]
Comment (drewnielson) [reply README.md:32]: Yes, that sounds better

**Comment by kengdoj** [comment]
See PR #212 for latest.

**Pull request closed without merging**